realm/config/findbugs/findbugs-filter.xml

```diff
@@ -21,5 +21,10 @@
     <Match>
         <Class name="io.realm.PermissionOfferResponseRealmProxy"/>
     </Match>
+    <Match>
```

Comment:
I'm pretty sure that you can just use an annotation for this: @SuppressFBWarnings("DLS_DEAD_LOCAL_STORE"). That would be a lot better: